src/get-started/quickstart.md

```diff
@@ -101,8 +116,8 @@ $ ls counts*
 ```
 
 {:.runner-flink}
-``` 
-TODO BEAM-899
+```
+$ ls counts*
```

Comment:
Please do as you see fit.

From my perspective, I'd point the users to it, but not teach them how to set up a cluster.